acceptance/bundle/sync/from-file/script

```diff
@@ -0,0 +1,21 @@
+mkdir "project-folder" "project-folder/blob" "ignored-folder" "ignored-folder/folder1" ".git"
+touch "project-folder/app.yaml" "project-folder/app.py" "project-folder/query.sql" "project-folder/blob/1" "project-folder/blob/2"
+touch "ignored-folder/script.py" "ignored-folder/folder1/script.py" "ignored-folder/folder1/script.yaml" "ignored-folder/folder1/big-blob"
+cat > .gitignore << EOF
+ignored-folder/
+script
+output.txt
+repls.json
+EOF
+cat > .databricksignore << EOF
+project-folder/blob/*
+project-folder/*.sql
+EOF
```

Comment:
any reason to create these from script rather than create it in your worktree and commit with git?

Reply:
+1, lets serialize test fixture files.